ocaml/typing/printtyp.ml

```diff
@@ -2428,37 +2290,15 @@ let class_type ppf cty =
   prepare_class_type [] cty;
   !Oprint.out_class_type ppf (tree_of_class_type Type [] cty)
 
-<<<<<<< HEAD
-let tree_of_class_param param var_inj =
```

Comment:
It looks like we're losing some logic here, but I did not look all that closely.

Reply:
I think we're not losing any logic here.